app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -114,8 +114,10 @@ def link_to_field(fieldname, fieldvalue, displayvalue = nil)
     end
 
     def iconify_auto_link(text, showLink = true)
+      # this block is only executed when a link is inserted;
+      # if we pass text containing no links, it just returns text.
       auto_link(text) do |value|
-        "<i class='glyphicon glyphicon-new-window'></i>&nbsp;#{value if showLink}<br />"
+        "<i class='glyphicon glyphicon-new-window'></i>#{"&nbsp;#{value}" if showLink}<br />"
```

Comment:
Nested `#{}`s almost never work. I think what you need here is something like `#{'(&nbsp;' + value) if showLink}`